src/components/AddContent/helpers.ts

```diff
@@ -99,41 +99,41 @@ export const makeValidationSchema = () => {
 };
 
 export const magicURLList = [
-  'https://www.google.com',
-  'https://apple.com',
-  'https://youtube.com',
-  'https://microsoft.com',
-  'https://play.google.com',
-  'https://linkedin.com',
-  'https://www.blogger.com',
-  'https://cloudflare.com',
-  'https://en.wikipedia.org',
-  'https://mozilla.org',
-  'https://docs.google.com',
-  'https://youtu.be',
-  'https://maps.google.com',
-  'https://wordpress.org',
-  'https://accounts.google.com',
-  'https://europa.eu',
-  'https://drive.google.com',
-  'https://whatsapp.com',
-  'https://adobe.com',
-  'https://sites.google.com',
-  'https://plus.google.com',
-  'https://facebook.com',
-  'https://vk.com',
-  'https://github.com',
-  'https://istockphoto.com',
-  'https://amazon.com',
-  'https://uol.com.br',
-  'https://policies.google.com',
-  'https://vimeo.com',
-  'https://t.me',
-  'https://mail.ru',
-  'https://enable-javascript.com',
-  'https://medium.com',
-  'https://search.google.com',
-  'https://google.de',
-  'https://ok.ru',
-  'https://imdb.com',
+  'http://mirror.stream.centos.org/9-stream/BaseOS/x86_64/os/',
```

Comment:
Not sure if the trailing "/" matter as you append /repodata/repomd.xml in the backend. This works though.

Reply:
it doesn't matter here because i'm using a path.Join() in the backend. 

Currently we treat trailing slash and no-trailing slash urls as unique urls though and i've been meaning to file a bug around that!

Reply:
Oh good point.